### ISSUE TITLE
Component demo copy code button provides no SR feedback when actioned

### ISSUE THREAD

**Issue by DBD324** [mi6/ic-design-system#613]
## Summary

When interacting with component demos using a SR(NVDA), no confirmation is announced when the button is clicked. When it is clicked a tooltip appears on screen saying 'Code copied to clipboard'. This content should be announced by a screen reader. 

## 💬 Description

Make tooltip content annouceable when a copy code button is clicked so that SR users receive some feedback confirmation on their action. 

## 💰 Use value

Providing feedback after a user action is important as it lets the user know if their action was performed correctly or not. This principle should be the same for SRs too. 

## 📝 Acceptance Criteria

If relevant, describe in full detail the different interactions and edge cases that the component or patterns needs to fulfil.

_Given_ that a user is navigating through a code guidance page using a SR
_When_  focussing and triggering a copy code button
_Then_ the SR should announce the contents on the tooltip that appears visually reading 'Code copied to clipboard.'

## ✏ Designs

If there's a Figma design file (or other mock-up), include it here.

## 🧾 Guidance
If there's written guidance or documentation, include a link to it here.

## Additional info

Tell us anything else useful to help us understand your suggestion.

**Comment by GCHQ-Developer-299** [comment]
@DBD324  we agreed in refinement that we should not be using a tooltip to confirm the action of a button. 
Would you agree that a Toast would be more appropriate? (as it would provide built in screenreader confirmation)
